src/test/java/cloud/fogbow/ras/core/processors/ClosedProcessorTest.java

```diff
@@ -60,34 +50,66 @@ public void tearDown() {
-        localOrder.setInstanceId(instanceId);
+        // exercise
+        this.thread = new Thread(this.processor);
+        this.thread.start();
+        Thread.sleep(BaseUnitTests.DEFAULT_SLEEP_TIME);
+
+        // verify
+        Mockito.verify(this.orderController, Mockito.times(1)).deactivateOrder(Mockito.eq(order));
+        Assert.assertNull(closedOrderList.getNext());
+        Assert.assertNull(activeOrdersMap.get(order.getId()));
+        Assert.assertEquals(OrderState.DEACTIVATED, order.getOrderState());
+    }
+    
+    // test case: Check the throw of UnexpectedException when running the thread in
+    // the closed processor, while running a local order.
+    @Test
+    public void testRunProcessLocalOrderThrowsUnexpectedException() throws InterruptedException, FogbowException {
+        // set up
+        Order order = createLocalOrder(getLocalMemberId());
+        this.closedOrderList.addItem(order);
 
-        this.orderController.activateOrder(localOrder);
-        OrderStateTransitioner.transition(localOrder, OrderState.CLOSED);
+        Mockito.doThrow(new UnexpectedException()).when(this.processor).processClosedOrder(Mockito.eq(order));
 
-        //exercise
-        this.thread = new Thread(this.closedProcessor);
+        // exercise
+        this.thread = new Thread(this.processor);
         this.thread.start();
-        Thread.sleep(500);
+        Thread.sleep(BaseUnitTests.DEFAULT_SLEEP_TIME);
 
-        //verify
-        SharedOrderHolders sharedOrderHolders = SharedOrderHolders.getInstance();
-        ChainedList<Order> closedOrders = sharedOrderHolders.getClosedOrdersList();
-        Map<String, Order> activeOrdersMap = sharedOrderHolders.getActiveOrdersMap();
-        assertNull(activeOrdersMap.get(localOrder.getId()));
+        // verify
+        Mockito.verify(this.processor, Mockito.times(1)).processClosedOrder(Mockito.eq(order));
+    }
+    
+    // test case: During a thread running in closed processor, if any
+    // errors occur, the processClosedOrder method will catch an exception.
+    @Test
+    public void testRunProcessLocalOrderToCatchException() throws InterruptedException, FogbowException {
 
-        closedOrders.resetPointer();
-        assertNull(closedOrders.getNext());
+        // set up
+        Order order = createLocalOrder(getLocalMemberId());
+        this.closedOrderList.addItem(order);
+
+        Mockito.doThrow(new RuntimeException()).when(this.processor).processClosedOrder(Mockito.eq(order));
+
+        // exercise
+        this.thread = new Thread(this.processor);
+        this.thread.start();
+        Thread.sleep(BaseUnitTests.DEFAULT_SLEEP_TIME);
 
-        //TODO: it would be good to verity that the OrderStateTransitioner.deactivateOrder was called. However,
-        //it is static thus, hard to mock
+        // verify
+        Mockito.verify(this.processor, Mockito.times(1)).processClosedOrder(Mockito.eq(order));
     }
+    
```

Comment:
there are exceptions declared in some method that are never thrown (the IDE can spot these mistakes if configured)

Reply:
I set up the IDE and now I could identify unused exception throws. 
Consider as done.